Software/build/IDEs/CCS/main.c

```diff
@@ -1,9 +1,157 @@
 
+#include "Can.h"
+
 
 /**
  * main.c
  */
+#define LOOPBACK           STD_ON
+
+void Can_Port_Enable(void);
+void PLL_Init(void);
+void PortF_Init(void);
+void Test1_RxTx_Polling(void) ;
+
+extern uint8 ReadData;
+uint8 WriteData;
+extern const Can_ConfigType Can_Configurations ;
+//static uint8 dataT[]={0x55,0x55,0x55,0x55,0x55,0x55,0x55,0x55} ;
+static Can_PduType  PduInfo=
+                            {
+                             1,
+                             1,     /*Length*/
+                             0x2,   /*ID*/
+                             &WriteData
+                            } ;
+
+//static uint8 counter=0;
+volatile uint32 SW1=0,SW2=0;
 int main(void)
 {
+
+    /*Set system clock to 80 MHz*/
+    PLL_Init();
+    PortF_Init();
+    Can_Port_Enable();
+    Can_Init(&Can_Configurations) ;
+    #if(LOOPBACK==STD_ON)
```

Comment:
What is the usage of loop back here?

Reply:
Just to debug c code that doesn't require the real bus